x-pack/plugins/embeddable_enhanced/kibana.jsonc

```diff
@@ -5,7 +5,7 @@
     "@elastic/kibana-presentation"
   ],
   "group": "platform",
-  "visibility": "private",
+  "visibility": "shared",
```

Comment:
@elastic/kibana-presentation this PR opens up `embeddableEnhanced` so that it is used directly from solutions (in this case from o11y SLOs).

Up until now, it was only used from "platform" plugins (e.g. _visualizations, dashboard_enhanced, lens, maps_).

Are you okay with that transition to "shared"?

Reply:
Yes this makes sense IMO. Solutions are expected to register embeddable types, and some of those will use drilldowns.